jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/HttpOutput.java

```diff
@@ -354,7 +354,7 @@ else if (closeContent != null)
         finally
         {
             if (wake)
-                _servletChannel.execute(_servletChannel); // TODO review in jetty-10 if execute is needed
+                _servletChannel.execute(_servletChannel::handle); // TODO review in jetty-10 if execute is needed
```

Comment:
Is this TODO still relevant, and shouldn't it be in jetty-10 code rather than jetty-12 code?